src/PeriodEnd/TransferPayments/DeployTransferPayments/sql/dml/PeriodEnd.TransferPayments.Cleanup.Deds.DML.sql

```diff
@@ -1,9 +1,7 @@
-DECLARE @CollectionPeriodMonth int = (SELECT [CalendarMonth] FROM Reference.CollectionPeriods WHERE [Open] = 1)
-DECLARE @CollectionPeriodYear int = (SELECT [CalendarYear] FROM Reference.CollectionPeriods WHERE [Open] = 1)
+DECLARE @CollectionPeriodName varchar(8) = (SELECT [AcademicYear] FROM Reference.CollectionPeriods WHERE [Open] = 1) + '-' + (SELECT [Name] FROM Reference.CollectionPeriods WHERE [Open] = 1)
+
 
 DELETE FROM ${DAS_PeriodEnd.FQ}.[Payments].[Payments]
-    WHERE 1=1
-        AND [CollectionPeriodMonth] = @CollectionPeriodMonth
-        AND [CollectionPeriodYear] = @CollectionPeriodYear
+    WHERE [CollectionPeriodName] = @CollectionPeriodName
         AND [FundingSource] = 5
 GO
```

Comment:
this statement appears to be deleting payments and did the bug also delete some transfer payments? Was this error destructive? Also FMI why do we even delete Funding Source 5 payments. I've not looked at this bit of code in details

Reply:
The whole period end/submission process ends with deleting the payments/datalocks/required payments/etc for the current period from deds (in case it's partially run before) and then copying to deds. These scripts are the delete part.

The error was destructive. It was brought to light by all the datalock validation errors for R13 disappearing after R02 was run